requirements/pip-tools.txt

```diff
@@ -12,7 +12,7 @@ packaging==21.3
     # via build
 pep517==0.13.0
     # via build
-pip-tools==6.8.0
+pip-tools==6.9.0
```

Comment:
No breaking changes in the new version: https://github.com/jazzband/pip-tools/blob/master/CHANGELOG.md#690-2022-10-05